### ISSUE TITLE
Add impact on accessibility

### ISSUE THREAD

**Issue by LJWatson** [w3c/ai-web-impact#24]
Generative AI is already having a systemic impact on accessibility on the Web, and W3C needs to understand the positive and negative effects it's having.

For example, generative AI is used in browsers, websites, and apps, to provide descriptions of images for anyone unable to see them.

Despite their [tendancy to hallucinate](https://tink.uk/adventures-with-bemyai/), the rapid evolution of LLM and tools like ChatGPT and Gemini, have effectively made any image on the Web accessible to blind people. To put this into context, according to the [WebAIM Million report](https://webaim.org/projects/million/) the homepages of the top one million websites have an average of 55.5 images and 21.6% of those are missing text descriptions.

The same technologies are also being used to sell products that claim to make websites instantly accessible. Jakob Nielsen takes it a step further, claiming that [accessibility has failed](https://jakobnielsenphd.substack.com/p/accessibility-generative-ui) in its current form and that generative AI will solve everything.

These products do [not accomplish](https://overlayfactsheet.com/en/) all they claim, and (I think) [Nielsen needs to think again](https://tink.uk/nielsen-needs-to-think-again/), but generative AI is constantly evolving and the appeal of instant and affordable accessibility is not going to go away.

As the organization at the forefront of accessibility on the Web for nearly 30 years, W3C undoubtedly has a role to play in understanding and responding to these systemic changes - and perhaps a certain amount of self-reflection about the role its played in creating a Web that has these gaps that generative AI is being used to close or worse, where generative AI is used to sell solutions that claim to close them with more confidence than they should.

**Comment by cynthia** [comment]
There is also the case to be made for different accessibility needs as well; speech to text transcription is already deployed in the wild which provides text descriptions for audio content. Cognitive accessibility needs could also be served through text models (e.g. please rephrase this in simple english, please summarize for two simple examples).

While both have limits on the guarantees in terms of bias and accuracy, there is definite promise for incremental progress for both cases.

**Comment by dontcallmedom** [comment]
for future reference (including my own since I haven't read it through yet) https://lists.w3.org/Archives/Public/w3c-wai-gl/2024AprJun/thread.html#msg5
